Source/ACE/Entity/WorldObject.cs

```diff
@@ -2695,5 +2726,50 @@ private void RecallAndSetPhysicsStateBools()
             if (AceObject.IsFrozen ?? false)
                 Frozen = true;
         }
+
+        public WorldObject GetObjectFromWeenieType()
```

Comment:
I don't understand this at all.  This creates a new object with the same weenie type and AceObject as the current object?  Why is this appropriate, and where is it used?